requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests>=2.20.0
-aiocometd>=0.4.5
-py_mini_racer>=0.1.18
+### aiocometd == 0.4.5 ### disable
+aiocometd @ git+https://github.com/random-python/x_aiocometd@develop ### enalbe ###
```

Comment:
Either tag it at the commit or leave this unfixed until it gets merged into a proper pypi release. I don't want to directly pull the latest version of a dev branch in case it randomly changes and breaks things.